blocks/subscriptions-block/themes/news.json

```diff
@@ -0,0 +1,301 @@
+{
+	"checkout": {
+		"styles": {
+			"default": {
+				"font-family": "var(--font-family-primary)",
+				"margin-bottom": "var(--global-spacing-5)",
+				"components": {
+					"heading": {
+						"font-size": "var(--global-font-size-12)"
+					},
+					"link": {
+						"color": "var(--color-primary)",
+						"border-bottom": "1px solid var(--color-primary)"
+					},
+					"link-hover": {
+						"color": "var(--color-primary-hover)",
+						"border-bottom": "1px solid var(--color-primary-hover)"
+					}
+				}
+			}
+		}
+	},
+	"checkout-cart": {
+		"styles": {
+			"default": {
+				"margin-top": "var(--global-spacing-6)",
+				"components": {
+					"heading": {
+						"margin-bottom": "var(--global-spacing-5)",
+						"font-size": "var(--global-font-size-11)"
+					}
+				}
+			}
+		}
+	},
+	"checkout-cart-items": {
+		"styles": {
+			"default": {
+				"border": "1px solid var(--global-neutral-3)"
```

Comment:
There's an alias `var(--border-color)` that should be used here instead this global color.